UnderSea.BLL/DTO/IdDTO.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace UnderSea.BLL.DTO
+{
+    public class IdDTO
```

Comment:
Erre miért van szükség?

Reply:
Márk (android) kérte, hogy megkönnyítse a munkáját.

Reply:
Azért van neki külön objektum, mert a frontendesek kérték, hogy egy elemet is objektumba csomagolva lehessen visszaküldeni.

Reply:
érdekes, de ok